lib/libzutil/zutil_import.c

```diff
@@ -1005,27 +1005,32 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 	}
 
 	if (lio_listio(LIO_WAIT, aiocbps, VDEV_LABELS, NULL) != 0) {
+		/*
+		 * A portion of the requests may have
```

Comment:
Now the comment is too high.  It should go just beneath the check for EAGAIN, EINTR, and EIO.

Reply:
Ah, I see, I'm sure I had good reason to put it there before, but looking at your prior statement now, I'm unsure how I could have misinterpreted it.

Fixed.